net-dns/libidn2/libidn2-2.0.4.recipe

```diff
@@ -0,0 +1,75 @@
+SUMMARY="Internationalized Domain Names (IDN) implementation"
+DESCRIPTION="Libidn2 is a free software implementation of IDNA2008, Punycode and TR46. Its \
+purpose is to encode and decode internationalized domain names."
+HOMEPAGE="https://gitlab.com/libidn/libidn2"
+COPYRIGHT="2011-2016 Simon Josefsson"
+LICENSE="GNU LGPL v3"
+REVISION="1"
+SOURCE_URI="http://ftp.gnu.org/gnu/libidn/libidn2-$portVersion.tar.gz"
+CHECKSUM_SHA256="644b6b03b285fb0ace02d241d59483d98bc462729d8bb3608d5cad5532f3d2f0"
+
+ARCHITECTURES="x86_gcc2 x86 x86_64"
+SECONDARY_ARCHITECTURES="x86_gcc2 x86"
+
+libVersion="0.3.3"
+libVersionCompat="$libVersion compat >= ${libVersion%%.*}"
+
+PROVIDES="
+	libidn2$secondaryArchSuffix = $portVersion
+	lib:libidn2$secondaryArchSuffix = $libVersionCompat
+	lib:libgettextlib_$portVersion$secondaryArchSuffix
+	texinfo = $portVersion compat >= 4
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	"
+
+PROVIDES_devel="
+	libidn2${secondaryArchSuffix}_devel
+	devel:libidn2$secondaryArchSuffix
+	"
+REQUIRES_devel="
+	libidn2$secondaryArchSuffix == $portVersion base
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	devel:libiconv$secondaryArchSuffix = $libVersionCompat
+	"
+BUILD_PREREQUIRES="
+	cmd:aclocal
+	cmd:autoconf
+	cmd:automake
+	cmd:autopoint
+	cmd:gcc$secondaryArchSuffix
+	cmd:ld$secondaryArchSuffix
+	cmd:libtoolize$secondaryArchSuffix
+	cmd:make
+	cmd:find
+	cmd:tar
+	cmd:gzip
+	git
+	perl
+	"
+
+BUILD()
+{
+	autoreconf -fi
+	runConfigure ./configure
+	make $jobArgs
+}
+
+INSTALL()
+{
+	make install
+
+	rm $libDir/libidn2.la
```

Comment:
```diff
-	rm $libDir/libidn2.la
+	rm -f $libDir/libidn2.la
```
(This is recommended in case newer versions of the tools stop creating the libtool files.)